src/pcapi/core/offers/api.py

```diff
@@ -119,9 +118,9 @@ def create_offer(offer_data: PostOfferBodyModel, user: User) -> Offer:
             isNational=product.isNational,
             extraData=product.extraData,
         )
-    elif feature_queries.is_active(
-        FeatureToggle.ENABLE_ISBN_REQUIRED_IN_LIVRE_EDITION_OFFER_CREATION
-    ) and offer_data.type == str(ThingType.LIVRE_EDITION):
+    elif FeatureToggle.ENABLE_ISBN_REQUIRED_IN_LIVRE_EDITION_OFFER_CREATION.is_active() and offer_data.type == str(
+        ThingType.LIVRE_EDITION
+    ):
```

Comment:
Tiens, au passage,  on pourraint inverser l'ordre des expressions dans la condition, pour éviter une requête SQL si le type d'offre n'est pas celui qu'on veut.

```suggestion
    elif offer_data.type == str(ThingType.LIVRE_EDITION) and FeatureToggle.ENABLE_ISBN_REQUIRED_IN_LIVRE_EDITION_OFFER_CREATION.is_active() 
```